examples/src/Syntax/InvalidCnfs/Config.hs

```diff
@@ -4,16 +4,16 @@ import LogicTasks.Config (
   BaseConfig(..),
   CnfConfig(..),
   )
-import Tasks.LegalCNF.Config (
-  LegalCNFConfig(..), checkLegalCNFConfig,
+import Tasks.LegalNormalForm.Config (
+  LegalNormalFormConfig(..), checkLegalNormalFormConfig,
   )
 import Test.Hspec
 import Util.VerifyConfig
 import Control.OutputCapable.Blocks (Language(German))
 
 -- Weight 0.33
-task07 :: LegalCNFConfig
-task07 = LegalCNFConfig
+task07 :: LegalNormalFormConfig
+task07 = LegalNormalFormConfig
   { cnfConfig = CnfConfig
```

Comment:
Ist es nicht etwas merkwürdig, wenn der Typ `LegalNormalFormConfig` jetzt bewusst keinen namentlichen Bezug zu KNFs mehr hat, aber darin dann doch ein Feld noch als `cnfConfig :: CnfConfig` benannt ist?

Reply:
Ich könnte natürlich einen separaten Typen für CNF und DNF einführen, die wären dann aber eins zu eins dieselben, bis auf `cnfConfig`vs `dnfConfig`. 

Wenn man das so implementiert, steht da aber bei der "LegalDNFConfig" `dnfConfig :: CNFConfig`. Das liegt daran, dass überall `CnfConfig` überall als Config für DNFs verwendet werden.

Lösung dafür wäre den Namen von `CnfConfig` zu `CnfDnfConfig` zu ändern. Dann müsste man aber auch `minClauseAmount`, etc. auch in `minClauseConAmount` umbenennen. Ich weiß nicht, ob das schöner ist.

Reply:
Das ist unter der Annahme, bei disjunktiver Normalform könne man nicht auch von Klauseln reden? Das würde ich nicht so sehen. Die meisten kennen "Klauselform" nur von "KNF", weil im Resolutionsverfahren so benutzt. Aber auch von einer DNF gibt es die Klauselform, und das ist völlig dual zu KNFs. Also muss man bei der Benennung nicht zwischen `Clause` und `Con` unterscheiden. Entsprechend gibt es vielleich auch für `CnfConfig` eine neutrale Alternative, die nicht unbedingt `Cnf` und `Dnf` erwähnen muss.

Reply:
Vielleicht irre ich mich, aber ich dachte, dass Frau König damals gesagt hatte, dass man nur bei einer Disjunktion von einer Klausel spricht.

Wie dem auch sei, ich habe den Namen von `CnfConfig` zu `NormalFormConfig` geändert.

Reply:
In Herr Hoppes Vorlesung war immer folgendes Bild aus Wikipedia:

![image](https://github.com/user-attachments/assets/75d9d8a9-2c45-466c-b833-e4b865e66cf5)

Darin werden sowohl die Teilterme der KNF als auch die Teilterme der DNF als "Klauseln" bezeichnet (siehe Tabellenheader).